src/pages/LandingPage/index.jsx

```diff
@@ -11,7 +12,7 @@ export const LandingPage = () => {
       <div className="div">
         <div className="top-nav-bar">
           <div className="div-wrapper">
-            <div className="text-wrapper">Job Quest</div>
+          <NavigationBar isLoggedin= "true"/>
```

Comment:
The logic for checking for whether a user is logged in for the navbar should come from checking the localStorage item for a key with the value.